### PR TITLE
Add tests for ModulemdModuleStream description

### PR DESCRIPTION
**Pull request by Norem80**
Copied python tests for Modulemd.ModuleStream.description into C as a Google Code-in 2019 Task.

### CODE REVIEW

File: modulemd/tests/test-modulemd-modulestream.c [hunk 1]
```diff
@@ -21,6 +21,9 @@
   "À϶￥🌭∮⇒⇔¬β∀₂⌀ıəˈ⍳⍴V)"                           \
   "═€ίζησθლბშიнстемองจึองታሽ።ደለᚢᛞᚦᚹ⠳⠞⠊⠎▉▒▒▓😃"
 
+#define MMD_TEST_DESC_LOCALE "C"
```
Comment (OrionStar25) [inline modulemd/tests/test-modulemd-modulestream.c]: I don't think this is necessary as "C" refers to the default locale. It's unlikely to change. I personally think the readability is being lost by such a huge macro, but I think we'll let this stay. 

@sgallagher What do you think?

Comment (Norem80) [reply modulemd/tests/test-modulemd-modulestream.c]: So should I just write "C" as an argument rather then create macro?

**Comment by Norem80** [comment]
Added that.

**Comment by sgallagher** [comment]
There are conflicts with the current master branch. Would you mind doing a `git rebase` and resolving them, please?

**Comment by sgallagher** [comment]
@Norem80 Please rebase this pull request.

**Pull request merged**